datahub-web-react/src/app/post/PostListColumns.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Typography } from 'antd';
+// import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { Maybe } from '../../types.generated';
+import PostItemMenu from './PostItemMenu';
+
+interface PostEntry {
+    title: string;
+    contentType: string;
+    description: Maybe<string>;
+    urn: string;
+}
+
+const PostTitleContainer = styled.div`
+    margin-left: 16px;
+    margin-right: 16px;
+    display: inline;
+`;
+
+export function PostListMenuColumn(handleDelete: (urn: string) => void) {
+    return (record: PostEntry) => (
+        <PostItemMenu title={record.title} urn={record.urn} onDelete={() => handleDelete(record.urn)} />
+    );
+}
+
+export function PostTitleColumn() {
+    return (record: PostEntry) => (
+        <span data-testid={record.urn}>
+            <PostTitleContainer>
+                <Typography.Text>{record.title}</Typography.Text>
+            </PostTitleContainer>
+        </span>
+    );
+}
+
+export function PostDescriptionColumn() {
+    return (record: PostEntry) => (
+        <span>
+            <PostTitleContainer>
+                <Typography.Text>{record.description}</Typography.Text>
+            </PostTitleContainer>
+        </span>
+    );
+}
+
+export function PostContentTypeColumn() {
```

Comment:
all of these look pretty much the same except for the data-testid added - could we make this a singular generic function? we could pass in an optional `dataTestId` as a prop if we wanted that and only wanted that on one of the column titles